python/python/bystro/api/cli.py

```diff
@@ -155,15 +155,15 @@ def load_state(state_dir: str) -> CachedAuth | None:
     return None
 
 
-def save_state(data: CachedAuth, state_dir: str, print_result=True) -> None:
+def save_state(data: CachedAuth, bystro_token_path: str, print_result=True) -> None:
     """
     Saves the authentication state to a file.
 
     Parameters
     ----------
     data : CachedAuth
         The data to save.
-    state_dir : str
+    bystro_token_path : str
```

Comment:
let's rename bystro_token_dir, to better communicate that this isn't a path to the token, but the directory where we want to store bystro credentials

Reply:
please also make the default for this DEFAULT_DIR